sonar-project.properties

```diff
@@ -14,3 +14,6 @@ sonar.python.coverage.reportPaths=coverage.xml
 
 # Encoding of the source code. Default is default system encoding
 #sonar.sourceEncoding=UTF-8
+
+# Exclusions from coverage
+sonar.coverage.exclusions=**/tests/**/*.*, **/migrations/**/*.*, **/backend/admin/*.*, **/__init__.py, **/jwt_auth/admin.py, **/jwt_auth/tests.py, **/urls.py, **/wsgi.py, **/manage.py, **/settings.py
```

Comment:
If we are ignoring the tests for resources now that we're (hopefully) done migrating data, we can ignore the resources folder as well

Reply:
good catch. I will add that.
I believe after all the exclusions, sonar is giving the right number for the overall test coverage, so I also have added a fail-under flag for 75%.